lsnms/nms.py

```diff
@@ -71,7 +71,8 @@ def _nms(
         keep.append(current_idx)
         to_consider[current_idx] = False
 
-    return np.array(keep)
+    keep = np.array(keep)
+    return np.argwhere(score_mask)[keep]
```

Comment:
```suggestion
    return np.argwhere(score_mask)[:, 0][keep]
```
Did not know about `argwhere` returning 2d array !